packages/block-library/src/comments-pagination-next/index.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Server-side rendering of the `core/comments-pagination-next` block.
+ *
+ * @package WordPress
+ */
+
+
+/**
+ * Applies the block attributes to the block markup without using a wrapper.
+ *
+ * @return string Returns the styles and classes defined on the block editor for the block.
+ */
+function add_next_comments_link_attributes() {
+	if ( ! function_exists( 'get_block_wrapper_attributes' ) ) {
+		return;
+	}
+	return get_block_wrapper_attributes();
+}
+add_filter( 'next_comments_link_attributes', 'add_next_comments_link_attributes' );
```

Comment:
Is this a good way to add the attributes to the `a` tag?
cc: @gziolo @ntsekouras

Reply:
I think I don't know the answer. Why do we need to use the filter here? Maybe we need a different implementation fo `get_next_comments_link` instead?

As far as I can tell `get_next_comments_link` works only with the singular page so it reads everything from the WordPress's page context. If we want to support custom settings for order, items per page, etc... then we need a custom implementation.

Reply:
I haven't followed all the discussions about where the comment blocks are going, but with this approach `get_next_comments_link` and using the global query, I think this is a good way to add the attributes. That's what I did for Query pagination blocks. The thing is though that you should add the filter inside the `render` function and then you have to `remove_filter` before exiting.

Reply:
This is the only way I found to provide HTML attributes without a wrapper, directly into the `<a>` tag.